.github/workflows/linux.yml

```
@@ -1,6 +1,13 @@
 name: Test on Linux
 
-on: [push, pull_request]
+on:
+  push:
+    # Disable this on our own internal PRs.
```

This comment doesn't seem correct. This will still run on our own PRs.

```suggestion
    # Prevent duplicate workflows on PRs from upstream branches
```